ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/FieldInfo.java

```diff
@@ -30,7 +30,8 @@
 public class FieldInfo {
 
   public enum FieldType {
-    SYSTEM, KEY
+    SYSTEM, // To be removed in the future. 0.9 saw this value no longer used.
```

Comment:
Kept `SYSTEM` around for now, so that using a later CLI with old server doesn't result in a deserialization error.

Not strictly necessary, but simple to do.